### PR TITLE
Don't show "unknown topic" when help message is displayed

### PR DESCRIPTION
**Pull request by locriacyber**
Remove the error "unknown topic" when typing `help api` and similar help commands.

### CODE REVIEW

**Comment by joshgoebel** [comment]
Please upload snap of the error case your solving here.

**Comment by locriacyber** [comment]
Curiously, it's not broken at https://tic80.com/create

On branch 6ef9fda:

[Uploading a.webm…]()

File: src/studio/screens/console.c [hunk 1]
```diff
@@ -3548,13 +3548,12 @@ static void onHelpCommand(Console* console)
         FOR(const struct Handler*, ptr, Handlers)
             if(strcmp(ptr->cmd, param) == 0)
             {
+                foundTopic = true;
                 ptr->handler(console);
                 break;
             }
 
-        if (foundTopic) {
-            commandDone(console);
```
Comment (joshgoebel) [inline src/studio/screens/console.c:3556]: The fix on 3651 looks good (setting the flag in a situation I missed in my patch), but what makes you think this needs to be removed?  I'm not sure this change is necessary.

Comment (locriacyber) [reply src/studio/screens/console.c:3556]: `commandDone` is called twice (one more time below).

**Comment by locriacyber** [comment]
Unrelated: The website is using a newer version (from the version number) of TIC80. Is that fix private?

File: src/studio/screens/console.c (hunk 1 above)
Comment (joshgoebel) [reply src/studio/screens/console.c:3556]: Ah yes, I see!

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**